exporter/tanzuobservabilityexporter/metrics_exporter.go

```diff
@@ -64,11 +61,10 @@ func newMetricsExporter(settings component.ExporterCreateSettings, c config.Expo
 	if !cfg.hasMetricsEndpoint() {
 		return nil, fmt.Errorf("metrics.endpoint required")
 	}
-	hostName, port, err := cfg.parseMetricsEndpoint()
-	if err != nil {
+	if _, _, err := cfg.parseMetricsEndpoint(); err != nil {
```

Comment:
Since `cfg.Metrics.Endpoint` is being used now is this function still necessary?

Reply:
This call to parseMetricsEndpoint() is no longer needed.

Reply:
Actually when I took this code out, tests started failing. We expect a port number to be included with the URL, and parseMetricsEndpoint checks for that.